workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/repository/EmployeeRepository.java

```diff
@@ -8,8 +8,19 @@
 import java.util.List;
 
 public interface EmployeeRepository extends JpaRepository<Employee, String> {
-    //  WS - The below is the code to return a sorted list by Function
-    @Query("SELECT a FROM Employee a ORDER BY a.function ASC, a.team ASC, a.employeeFullName ASC")
+
+    @Query("SELECT a FROM Employee a ORDER BY a.function ASC, a.team ASC, a.employeeSurname ASC")
     List<Employee> findByFunction();
+
+    @Query("SELECT DISTINCT f.function FROM Employee f ORDER BY f.function ASC")
+    List<String> fieldSelect();
```

Comment:
The function name has to reflect what its exactly trying to do. So far unless I see the query  in the repository its unclear that the method is getting a list of unique functions.

Reply:
Changed to getUniqueFunctions()

Reply:
Chnaged again, this time to getListOfUniqueFunctions()